q2_moshpit/eggnog/tests/test_dbs.py

```diff
@@ -36,6 +36,23 @@ def test_fetch_eggnog_db(self, subp_run):
         ]
         subp_run.assert_called_once_with(cmd, check=True)
 
+    @patch('tempfile.TemporaryDirectory')
+    @patch("q2_moshpit.eggnog._dbs._download_and_build_hmm_db")
+    @patch("q2_moshpit.eggnog._dbs._download_fastas_into_hmmer_db")
+    @patch("q2_moshpit.eggnog._dbs._validate_eggnog_hmmer_taxon_id")
+    def test_fetch_eggnog_hmmer_db(
+        self, mock_validate, mock_fastas, mock_build, tmpdir
+    ):
+        tmpdir.return_value.__enter__.return_value = "tmp"
+        taxon_id = 1
+        mock_build.return_value = (1, 2, 3)
+
+        _, _, _, _ = fetch_eggnog_hmmer_db(taxon_id)
```

Comment:
You don't need to assign the outputs if you don't care about them.
```suggestion
        fetch_eggnog_hmmer_db(taxon_id)
```